Services/UserService/Interfaces/ICreateCommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace UserService.Interfaces
+{
+    public interface ICreateUser<in Input, out Output>
+    {
+        public Output Execute(Input email, Input password);
+    }
+}
```

Comment:
why we use generic to create conkrect object user?

Reply:
Do we need such interface at all? Why not ICommand?

Reply:
Yes, I think . We still have positive elements in using interface. Spartak said that both of ways are okay. I don't sure if we wouldn't have the same signature in Create / Edit / Delete User. I want to be sure that my code wouldn't fail anything else. If it would become like we have different arguments in all actions we change it to ICommand

Reply:
And a lot of code with it, all our 100 and more unit tests, injections, signatures in all methods. Yes really good idea to change it latter ;-)